nemo/collections/nlp/modules/common/megatron/attention.py

```diff
@@ -67,11 +67,11 @@
 
 try:
     # Flash Attention Triton
-    import pkg_resources
+    from importlib.metadata import version
     from flash_attn.flash_attn_triton import flash_attn_func as flash_attn_func_triton
 
     # pinned triton version for flash-attention triton https://github.com/HazyResearch/flash-attention/blob/main/flash_attn/flash_attn_triton.py#L3
-    assert pkg_resources.get_distribution("triton").version == '2.0.0.dev20221202'
+    assert version("pytorch-triton") == '2.0.0.dev20221202'
```

Comment:
This assertion fails when latest pytorch-trition is installed based on requirements.txt